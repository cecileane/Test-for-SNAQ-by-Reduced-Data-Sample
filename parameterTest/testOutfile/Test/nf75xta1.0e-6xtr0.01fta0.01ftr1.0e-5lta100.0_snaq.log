optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.01,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 75, liktolAbs = 100.0.
rootname for files: nf75xta1.0e-6xtr0.01fta0.01ftr1.0e-5lta100.0_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 64788
seed: 64788 for run 1
Sat Oct  1 17:15:21 2016

 BEGIN SNaQ for run 1, seed 64788 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 96 (failures: 75) with -loglik=126.74251 and ht_min=[0.2856,0.48942,0.51916,10.0,1.51213]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 11		 11		 1	 |	 NA 		 NA 		 NA 
mvorigin	 25		 25		 0	 |	 0		 0		 --
mvtarget	 21		 21		 0	 |	 0		 0		 --
chdir		 4		 4		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 35		 31		 0	 |	 4		 4		 --
Total		 96		 92		 1	 |	 4		 4		 4
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 126.74250981975207
((4,(2,(1)#H7:::0.7143969375956583):1.5121335797917066):0.4894189703316429,((6,#H7:::0.2856030624043418):10.0,5):0.5191607521958493,3);
---------------------
seed: 603 for run 2
Sat Oct  1 17:15:26 2016

 BEGIN SNaQ for run 2, seed 603 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 112 (failures: 75) with -loglik=227.80789 and ht_min=[0.18717,0.63689,0.48201,7.34651,0.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 9		 9		 1	 |	 NA 		 NA 		 NA 
mvorigin	 25		 25		 0	 |	 0		 0		 --
mvtarget	 25		 25		 0	 |	 0		 0		 --
chdir		 4		 4		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 49		 28		 0	 |	 0		 0		 --
Total		 112		 91		 1	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 2, -loglik of best 227.80788795330827
(3,((4,2):1.7699764903901208e-8,(1)#H7:::0.8128278510085798):0.6368931155925531,(5,(6,#H7:::0.18717214899142018):7.34650674917098):0.48200992101640167);
---------------------

Sat Oct  1 17:15:32 2016
MaxNet is ((4,(2,(1)#H7:::0.7143969375956583):1.5121335797917066):0.4894189703316429,((6,#H7:::0.2856030624043418):10.0,5):0.5191607521958493,3); 
with -loglik 126.74250981975207
