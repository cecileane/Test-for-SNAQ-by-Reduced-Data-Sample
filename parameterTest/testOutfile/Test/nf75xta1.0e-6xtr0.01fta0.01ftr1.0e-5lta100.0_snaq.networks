((4,(2,(1)#H7:::0.7143969375956583):1.5121335797917066):0.4894189703316429,((6,#H7:::0.2856030624043418):10.0,5):0.5191607521958493,3);, with -loglik 126.74250981975207 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
((4,(2,(1,#H7:::0.32573802122271334):2.3812298422815874):0.6613139920351059):0.45874830789685395,((6)#H7:::0.6742619787772867,5):2.006584817651667,3);, with -loglik 128.4983320490713
((4)#H7:::0.6285639111559405,((6,(1,(2,#H7:::0.3714360888440595):10.0):0.7348204158766389):0.2666827529928305,5):0.03869216941456422,3);, with -loglik 185.7011795966155
((4,#H7:::0.33556020284429766):3.8018890368348175,((6,(1,(2)#H7:::0.6644397971557023):1.2570051646496911):0.1486103832293508,5):0.0,3);, with -loglik 204.79990092381172
(#H7:::0.39145699038037607,(6,(1,(2,((3)#H7:::0.6085430096196239,4):0.08951533740410217):0.08326874576730896):0.8727163392445478):9.967577779809089,5);, with -loglik 216.57840037689357
((4,(2,(1,((5)#H7:::0.6341634939874272,6):0.7480185940077019):0.049408466375717094):0.0986574624850868):1.8474591263503386,#H7:::0.3658365060125728,3);, with -loglik 264.64992155359323
