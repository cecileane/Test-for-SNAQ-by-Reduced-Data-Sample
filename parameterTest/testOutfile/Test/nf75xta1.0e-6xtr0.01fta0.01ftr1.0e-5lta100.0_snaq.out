((4,(2,(1)#H7:::0.7143969375956583):1.5121335797917066):0.4894189703316429,((6,#H7:::0.2856030624043418):10.0,5):0.5191607521958493,3); -Ploglik = 126.74250981975207
 Dendroscope: ((4,(2,(1)#H7):1.5121335797917066):0.4894189703316429,((6,#H7):10.0,5):0.5191607521958493,3);
 Elapsed time: 10.86498657 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((4,(2,(1)#H7:::0.7143969375956583):1.5121335797917066):0.4894189703316429,((6,#H7:::0.2856030624043418):10.0,5):0.5191607521958493,3);, with -loglik 126.74250981975207
 (3,((4,2):1.7699764903901208e-8,(1)#H7:::0.8128278510085798):0.6368931155925531,(5,(6,#H7:::0.18717214899142018):7.34650674917098):0.48200992101640167);, with -loglik 227.80788795330827
-------
